aat/src/aat/resources/features/F-102 - TTL Base Complex Type/S-102.1.td.json

```diff
@@ -0,0 +1,29 @@
+{
+	"_guid_": "S-102.1",
+	"_extends_": "F-102_Test_Base_data",
+	"title": "must successfully import definition file",
```

Comment:
Title in scenario TD files should match feature file's corresponding scenario description.